.github/workflows/pythonpackage.yml

```diff
@@ -0,0 +1,33 @@
+name: Python 3 testing
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      max-parallel: 1
+      matrix:
+        python-version: [3.8]  # [2.7, 3.5, 3.6, 3.7]
```

Comment:
Why not 3.6 and 3.7 enabled ?

Reply:
They certainly could be.  Don't want to generate too much global warming in our testing. ;-)

Reply:
There already many warning in our codebase. I think it's fine :)